frontend/appflowy_flutter/lib/plugins/database_view/grid/presentation/widgets/header/type_option/number.dart

```diff
@@ -188,7 +181,9 @@ class NumberFormatCell extends StatelessWidget {
   Widget build(BuildContext context) {
     Widget? checkmark;
```

Comment:
Could you make this a getter?

```dart
Widget? get checkmark {
  if (isSelected) {
      checkmark = const FlowySvg(
        name: 'grid/checkmark',
      );
  }
  return null;
}
```

Reply:
I think the current way is fine too. The` rightIcon` in the `FlowyButton` can be null anyway.